src/components/Button/Button.tsx

```diff
@@ -1,24 +1,16 @@
+import { lightGrey, lighterBlue, mediumBlue, mediumGrey } from "../../const/styles";
+
 import styled from "styled-components";
-import {
-  lighterBlue,
-  lightGrey,
-  mediumBlue,
-  mediumGrey,
-} from "../../const/styles";
 
 interface ButtonProps {
-  onClick: () => void;
+  onClick?: () => void;
```

Comment:
kodėl padarei optional?

Reply:
Nes nepavyko apeiti log in checko kitu būdu, kitaip iššaudavo get requestą 2 kartus.